### PR TITLE
138323285-Add_application_close_date_to_framerwork

### PR DESCRIPTION
**Pull request by benvand**
#### 138323285-Add_application_close_date_to_framerwork

* Add application_close_date to framework model
* Update existing test

### Requires https://github.com/alphagov/digitalmarketplace-api/pull/531

### CODE REVIEW

File: migrations/versions/840_add_application_close_date_for_framework.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+"""Add `application_close_date` for Framework
```
Comment (TheDoubleK) [inline migrations/versions/840_add_application_close_date_for_framework.py]: The migration should go in a separate PR from the code, so it can be released on its own.

File: migrations/versions/840_add_application_close_date_for_framework.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+# revision identifiers, used by Alembic.
+revision = '840'
+down_revision = '830'
+
+
+
+def upgrade():
+    op.add_column('frameworks', sa.Column('application_close_date', sa.Date(), nullable=True))
```
Comment (TheDoubleK) [inline migrations/versions/840_add_application_close_date_for_framework.py]: All our other timestamps are `sa.DateTime()` - I know we only need the Date for the current use-case, but feels like it wouldn't hurt to keep it the same here.

In terms of the time of day to set - I think DOS closed at 3pm, G8 and DOS2 at 5pm.

File: app/models.py [hunk 3]
```diff
@@ -113,21 +118,19 @@ def get_lot(self, lot_slug):
         )
 
     def serialize(self):
+        acd = self.application_close_date.strftime(DATE_FORMAT) if self.application_close_date else ''
```
Comment (TheDoubleK) [inline app/models.py]: This could just be in-lined in the serialisation?  It's only used once, and everything else is in-line, so should be fine?

**Comment by benvand** [comment]
New PR for these changes to clean up commits:
https://github.com/alphagov/digitalmarketplace-api/pull/539
